src/MoBi.Core/Services/SimulationLoader.cs

```diff
@@ -69,13 +74,33 @@ private void addSimulationToProject(IMoBiSimulation simulation, MoBiMacroCommand
          addSimulationConfigurationToProject(moBiSimulation, loadCommand);
 
          moBiSimulation.ResultsDataRepository = simulation.ResultsDataRepository;
-         if (!_nameCorrector.CorrectName(project.Simulations, moBiSimulation))
-            return;
 
          moBiSimulation.HasChanged = true;
          loadCommand.AddCommand(new AddSimulationCommand(moBiSimulation));
       }
 
+      private void renameColidingModules(IReadOnlyList<Module> modulesToRename, string simulationName, IReadOnlyList<Module> existingModules, string originalSimulationName)
```

Comment:
In this case we aren't only renaming colliding modules so let's just take that out of the method name. Maybe this method should be called `correctModuleNames` because there is two types of correction happening